### DISCUSSION TITLE
architecture.md beeline example may be not correct

### DISCUSSION THREAD

**Discussion by zhenjiaguo** [apache/kyuubi#1337] [General]
https://github.com/apache/incubator-kyuubi/blob/96f843c3c0ad4e2d796788f4a3c842e54702e475/docs/overview/architecture.md?plain=1#L122-L134

I know that some spaces are redundant. But what I am not sure about is whether single quotes can work elsewhere. My system is Ubuntu, and using single quotes will cause errors. Double quotes will not.

**Comment by pan3793** [comment]
Thanks for verification and reporting this issue, got the same result on macOS with zsh/bash, would you like to send a PR to fix it?

**Reply by zhenjiaguo** [reply]
Yes, I’d like to send a PR to fix it.
